tracker/tracker.go

```diff
@@ -220,9 +221,13 @@ func (t *myTracker) run(stream gostream.VideoStream, cancelableCtx context.Conte
 				continue
 			}
 			filteredDets := FilterDetections(t.chosenLabels, detections, t.minConfidence)
+
 			// all new tracks get a fresh persistence counter
 			filteredNew := newTracks(filteredDets, t.minTrackPersistence)
 
+			// Here we will classify the cropped pizza detections and add that to the label
+			classifiedNew := classifyTracks(cancelableCtx, filteredNew, img, t.pizzaClassifier)
```

Comment:
There is an edge case I'm worried about -- let's say that one track had an error for the classifier, or all tracks had an error with the classifier, so therefore, all (or some) of these tracks have empty classification fields. I want you to make a test to see if this runs without error if some tracks have missing classification fields.